news/2021-05-13-mappers-report-april.md

```diff
@@ -0,0 +1,172 @@
+
+Oh, if you're a drum basher, fruit catcher or keyboard masher, this one is just for you! This year, the osu!taiko, osu!catch and osu!mania editions of Beatmapping World Championship are going to be organized as well! The details for these contests are work-in-progress, but you still have a chance to partake in the judging panels if you're interested in helping out - check out [this form](https://forms.gle/Yu2M2ZiwhapJYc7K7) for more details!
+
+The particulars concerning the contest and so much more can be found in full within [this newspost](https://osu.ppy.sh/home/news/2021-05-02-osu-beatmapping-world-championship-2021-registrations-open), the [wiki article](/wiki/Contests/oBWC/3) and the [official website](https://obwc.net/). Check them all out!
+
+### osu!mania J-Rock Mapping Contest
+
+Attention to all J-Rock fans out there! If you're keen on playing a keytar as well, this will probably catch your interest.
+
+Organized by [RiP46](https://osu.ppy.sh/users/5222596), osu!mania J-Rock Mapping Contest is featuring three lovely songs, each of which everyone can participate in.
+
+As for the most exciting part, the prizes, the winning mapset for each song will be pushed to ranked. Yes, you read it right, but that's not even it! The second and third place runner-ups will have the chance to participate in the winning set with a guest difficulty.
+
+Well, what are you waiting for? Start tuning your keytar and head over to the [forum thread](https://osu.ppy.sh/community/forums/topics/1312702) for details!
+
+### osu!mania Otaku TV Size Mapping Contest #2
+
+Looks like another osu!mania-related mapping contest is on the horizon. Here comes the Otaku TV Size Mapping Contest for you to enjoy the spirit of anime to the fullest!
+
+Hosted by [Sun](https://osu.ppy.sh/users/4115819), [Murumoo](https://osu.ppy.sh/users/8001433), [Paran](https://osu.ppy.sh/users/13340203) and [deta5859](https://osu.ppy.sh/users/15275662), the contest allows everyone to participate in each of the pre-selected two songs.
+
+The winning entry hosting the runner-ups' guest difficulties will be speed-carried to the ranked section, and you will have the pleasure of being remembered as an *Otaku Mapper*! For better or worse.
+
+If you're an anime fan, don't let this opportunity pass you by! Have a thorough look at this [forum thread](https://osu.ppy.sh/community/forums/topics/1308172) and you're good to go!
+
+<a id="conclusion"></a>
+
+---
+
+And that’s it for this month's issue of the Mappers’ Report! We are a bit late this month because most of our helpers were pretty busy with real life shenanigans. We can’t promise that the next issues will be on time, but we'll try our best!
```

Comment:
```suggestion
And that’s it for this month's issue of the Mappers’ Report! We were a bit late this month because most of our helpers were pretty busy with real life shenanigans. We can’t promise that the next issues will be on time, but we'll try our best!
```